### DISCUSSION TITLE
How to set the default text in Input widget

### DISCUSSION THREAD

**Discussion by Tim-Cao** [Textualize/textual#3165] [Q&A] [answered]
I would like to set a default text in Input widget

For example
The default env is Tomcat 9.0 + MysQL

```
                yield Label("Enter the test environment: ")
                yield Input(id='env')

```

**Comment by davep** [comment, marked as answer]
See the `value` parameter [in the docs](https://textual.textualize.io/widgets/input/#textual.widgets.Input).
